components/processing/form_parser/src/load_data_in_bigquery.py

```diff
@@ -28,8 +29,10 @@ def load_rows_into_bigquery(rows_to_insert, table_id):
         rows_to_insert: list of rows with each row as json string,
         table_id: bigquery table id
     """
+  USER_AGENT = "cloud-solutions/eks-agent-builder-v1"
```

Comment:
Please use the following value for the USER_AGENT string:
"cloud-solutions/eks-docai"

Reply:
updated